Gemfile

```diff
@@ -5,6 +5,9 @@ gem 'rubocop', '>= 1.0', '< 2.0'
 
 gem 'composite_primary_keys', '~>14.0'
 
+# handle Authorization
+gem 'cancancan'
+
```

Comment:
Nice, adding comments to the Gem file is something I've never done… I'll try to overcome my laziness and start adding them. Very informative.